src/Be.Vlaanderen.Basisregisters.Aws.DistributedMutex/DistributedLock.cs

```diff
@@ -104,13 +104,17 @@ public DistributedLock(DistributedLockOptions options, string lockName, ILogger
             _logger = logger;
             _lockName = lockName;
 
-            _mutex = CreateDynamoDbMutex(options);
+            _mutex = CreateMutex(options);
+            if (_mutex == null)
+            {
+                throw new NullReferenceException($"{nameof(CreateMutex)} result can't be null");
+            }
 
             _renewLeaseTimer.Interval = options.LeasePeriod.TotalMilliseconds / 2;
             _renewLeaseTimer.Elapsed += (sender, args) => RenewLease();
         }
 
-        private static DynamoDBMutex CreateDynamoDbMutex(DistributedLockOptions options)
+        protected virtual IMutex CreateMutex(DistributedLockOptions options)
```

Comment:
why virtual instead of static?

Reply:
so I can override in WR to be able to test it locally with a different implementation, since running the dynamodb local docker doesn't seem to work

Reply:
> running the dynamodb local docker doesn't seem to work

but it does work

Reply:
change reverted